src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,60 @@
-export const Pagination = () => {};
+// Pagination.tsx
+import React from 'react';
+
+interface PaginationProps {
+  pages: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  pages,
+  currentPage,
+  setCurrentPage,
+}) => {
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => setCurrentPage(currentPage > 1 ? currentPage - 1 : 1)}
```

Comment:
Using `href` with hash values can lead to unwanted behavior in single-page applications as it may cause the page to jump. Consider using `button` elements instead of `a` elements for pagination controls, or prevent the default action in the `onClick` handler.